yellowbrick/features/pcoords.py

```diff
@@ -1,28 +1,29 @@
 # yellowbrick.features.pcoords
 # Implementations of parallel coordinates for feature analysis.
 #
-# Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
-# Created:  Mon Oct 03 21:46:06 2016 -0400
+# Author:  Benjamin Bengfort <bbengfort@districtdatalabs.com>
+# Author:  @thekylesaurus
+# Created: Mon Oct 03 21:46:06 2016 -0400
 #
 # Copyright (C) 2016 District Data Labs
 # For license information, see LICENSE.txt
 #
 # ID: pcoords.py [0f4b236] benjamin@bengfort.com $
 
 """
-Implementations of parallel coordinates for multi-dimensional feature
-analysis. There are a variety of parallel coordinates from Andrews Curves to
```

Comment:
Reminder to check to see if we still have an open issue to implement Andrews Curves as an independent visualizer, and if so, consider the extent to which these fast/slow issues may also be relevant there.

Reply:
Yep - #100; I'll add a comment about fast and slow referencing this PR.